### DISCUSSION TITLE
Colors generator generating tuple without the EXACT base color I provided

### DISCUSSION THREAD

**Discussion by Mislav-Markusic** [mantinedev/mantine#6088] [Q&A]
This has happened to me quite a few times already, I don't really recall the previous colors I've tried, but the last one is: #2A4152
Here is the array I get from it:
`[
  "#f2f5f8",
  "#e4e7ea",
  "#c5cfd6",
  "#a2b4c2",
  "#859eb1",
  "#7290a7",
  "#6789a4",
  "#57768f",
  "#4b6980",
  "#3b5b73"
]`

And the colors I get are waaaay of, the last one is the darkest in the tuple, and here you can see direct comparison with my "base" color on the left, and #3b5b73 on the right
![Screenshot_20240415_213750](https://github.com/mantinedev/mantine/assets/117269017/3ed0aca8-651b-408b-8864-3d3d2f3140fd)

Am I doing something wrong here or is the colors generator broken / unusable?

**Comment by rtivital** [comment]
It works as intended. Colors generator is built to generate colors that have close luminosity to default Mantine colors. Your exact color is not guaranteed to be in the list.

**Reply by Mislav-Markusic** [reply]
First of all, thanks for the quick response

Shouldn't closest luminosity to my base color be somewhere in the middle of the array though, like array[6]? This way darkest color is still lighter than my color

**Reply by rtivital** [reply]
No, colors generated by the tool, have close luminosity to default Mantine colors to try to provide a proper contrast in all color schemes and components variants. If you want to use specific colors in your application, you can ask a designer to generate a 10 shades palette or use a different tool.
